<ui version="4.0" >
 <class>EnvListEditorBase</class>
 <widget class="QWidget" name="EnvListEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EnvVar List Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="5" >
    <widget class="QToolButton" name="mRemoveValueBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QToolButton" name="mAddValueBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3" >
    <widget class="QTableView" name="mValuesTable" />
   </item>
   <item row="0" column="3" >
    <widget class="QLabel" name="mValuesLbl" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>4</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Values:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QToolButton" name="mRemoveKeyBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="mAddKeyBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="mKeyLbl" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>4</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Key:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QListView" name="mKeysList" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../MaestroResource.qrc" />
 </resources>
 <connections/>
</ui>
